src/main/java/io/kestra/plugin/dbt/cli/AbstractDbt.java

```diff
@@ -56,7 +57,7 @@ public abstract class AbstractDbt extends AbstractBash implements RunnableTask<S
 
     @Builder.Default
     @Schema(
-        title = "The path to dbt cli"
+        title = "The path to the dbt cli"
```

Comment:
```suggestion
        title = "The path to the dbt project directory"
```

might be more specific